pkg/daemon/controller/remote.go

```diff
@@ -0,0 +1,177 @@
+package controller
+
+import (
+	"fmt"
+	"net"
+	"reflect"
+
+	ramav1 "github.com/oecp/rama/pkg/apis/networking/v1"
+	"github.com/oecp/rama/pkg/daemon/containernetwork"
+	"github.com/vishvananda/netlink"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/klog"
+)
+
+type RemoteVtepCache struct {
+	remoteVtepIPMap map[string]net.HardwareAddr
+
+	ch chan struct{}
+}
+
+func NewRemoteVtepCache() *RemoteVtepCache {
+	return &RemoteVtepCache{
+		remoteVtepIPMap: map[string]net.HardwareAddr{},
+		ch:              make(chan struct{}, 1),
```

Comment:
buffered channel is Non-blocking, this lock can't play a role. Use sync.RWMutex{} instead.

Reply:
I just follow the code style of NodeCache, but I think sync.RWMutex is a good idea.

Reply:
which nodecache?

Reply:
`NodeIPCache` in `pkg/daemon/controller/node.go`. Anyway, I shall take your advice.

Reply:
ok